lib/ftu/rootpage.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+
+import '../Navigation/app_controller.dart';
+import './authentication.dart';
+import './signuplogin.dart';
+
+enum AuthStatus {
+  notDetermined,
+  notLoggedIN,
```

Comment:
Little typo with the `IN`. Should be `In`.

Reply:
Oh... thanks